### PR TITLE
Add initial base project docs

### PR DESCRIPTION
**Pull request by heliocastro**
Add missing README, CODE_OF_CONDUCT, CONTRIBUTING

### CODE REVIEW

**Comment by KoukiHama** [comment]
FYI : this is also code of conduct in sw360 
https://github.com/eclipse-sw360/sw360/blob/main/CODE_OF_CONDUCT.md

**Comment by heliocastro** [comment]
@KoukiHama Sorry, need request approval again ( rebase )

**Comment by KoukiHama** [comment]
@ag4ums Could you also check this PR ?

README, CODE_OF_CONDUCT, CONTRIBUTING are very import for oss project.
Without both Co Leaders approve, we can merge it.

@heliocastro 
I would like to merge as soon as possible, but let's wait and see a bit.
We may be in the minority, checking GitHub over the weekend as well .

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (KoukiHama) [inline CODE_OF_CONDUCT.md:64]: Is [INSERT CONTACT METHOD] sw360-dev@eclipse.org?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# SW360 Frontend project contributing guide
+
+This is your first stop to contribute to SW360 Frontend project. From here you will find an overview of our contribution workflow.
+
+Please read our code [Code of Conduct](./CODE_OF_CONDUCT.md) before to help with the well being of the community.
+
+## New contributor guide
+
+To get an overvview of the project, read the [README](./README.md).
```
Comment (KoukiHama) [inline CONTRIBUTING.md:9]: typo:  overview

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+NEXTAUTH_SECRET = _any string_ (next auth secret key)
+```
+
+Then run:
+
+```bash
+> npm run dev
+```
+
+If everything went well, a port 3000 wil be open in your machine and you will able to access the app at the following address:
```
Comment (KoukiHama) [inline CONTRIBUTING.md:39]: typo : wil → will

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+the community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].
+
+Community Impact Guidelines were inspired by 
+[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
```
[the remark this answers is not in the document]
Comment (heliocastro) [reply CODE_OF_CONDUCT.md:123]: Agree on reuse the sw360 one. Still valid there's no reason to go different

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (heliocastro) [reply CODE_OF_CONDUCT.md:64]: Outdated as we will use the sw360 one.

File: CONTRIBUTING.md (hunk 3 above)
Comment (heliocastro) [reply CONTRIBUTING.md:39]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (heliocastro) [reply CONTRIBUTING.md:9]: Fixed

**Review by KoukiHama — APPROVED** [review]

**Comment by ag4ums** [comment]
@KoukiHama, @heliocastro why we need to change the header format in contribution guideline from sw360 to sw360-frontend, we should use the resue-complient traditional header format.

**Pull request merged**